### PR TITLE
Add `--merge` flag to `cluster kubeconfig` command.

### LINKED ISSUE #71: Consider cloudctl cluster kubeconfig to merge kubeconfig
It would make more sense if the `cluster kubeconfig` command does not print the kubeconfig but merge it to the path specified by `KUBECONFIG` / default kubernetes kubeconfig path. This would be more similar to how other tools deal with this issue, too.

### PR DESCRIPTION
**Pull request by Gerrit91**
Closes #71.

## Breaking Change

```BREAKING_CHANGE
The --kubeConfig parameter was renamed to --kubeconfig in order to align with kubectl.
```

### CODE REVIEW

File: cmd/cluster.go [hunk 1]
```diff
@@ -666,13 +673,71 @@ func clusterKubeconfig(args []string) error {
...
+	auth.SetCurrentContext(currentCfg, contextName)
+
+	mergedKubeconfig, err := yaml.Marshal(currentCfg)
 	if err != nil {
 		return err
 	}
 
-	// print kubeconfig
-	fmt.Println(string(mergedKubeconfig))
+	err = ioutil.WriteFile(filename, mergedKubeconfig, 0600)
```
Comment (majst01) [inline cmd/cluster.go:734]: os.Writefile

File: cmd/cluster.go [hunk 2]
```diff
@@ -666,13 +669,38 @@ func clusterKubeconfig(args []string) error {
...
 	}
 
-	// print kubeconfig
-	fmt.Println(string(mergedKubeconfig))
+	clusterResp, err := cloud.Cluster.FindCluster(cluster.NewFindClusterParams().WithID(id), nil)
+	if err != nil {
+		return err
+	}
+
+	mergedKubeconfig, err := helper.MergeKubeconfigTpl(kubeconfigTpl, currentCfg, *clusterResp.Payload.Name, authContext)
```
Comment (majst01) [inline cmd/cluster.go:692]: what if clustername  clashes with existing context ?

Comment (Gerrit91) [reply cmd/cluster.go:692]: It always updates and ... I was also thinking about longer context names like `<tenant>-<project-name>-<cluster-name>` but then it did not look so good. I think when a user decides to merge kubeconfigs together, he can assume that existing entries will be updated.

Comment (majst01) [reply cmd/cluster.go:692]: really ?

**Comment by majst01** [comment]
please fix metal-lib dependency

**Comment by Gerrit91** [comment]
metal-lib is now on v0.8.2.

**Review by majst01 — APPROVED** [review]

**Pull request merged**